docs/getting-started.md

```diff
@@ -67,19 +67,25 @@ directory (default: `target/${toolchain}/${mode}`, where mode can be `debug` or
 To build a kernel for the reference VMM to boot, check out the scripts in
 [resources/kernel](../resources/kernel).
 
-- [`make_busybox.sh`](../resources/kernel/make_busybox.sh) builds an ELF image
-  with a baked-in initramfs running [Busybox](https://busybox.net/). It uses a
-  stripped-down
+- [`make_kernel_image.sh`](../resources/kernel/make_kernel_image.sh) builds an
+  ELF or bzImage kernel with a baked-in initramfs running
+  [Busybox](https://busybox.net/). It uses a stripped-down
   [kernel config](../resources/kernel/microvm-kernel-initramfs-hello-x86_64.config)
   and a statically linked [config](../resources/kernel/busybox_static_config)
   for the Busybox initramfs.
 
+  For example:
+
   ```bash
-  ./make_busybox.sh
+  ./make_kernel_image -f elf -k vmlinux-hello-busybox -w /tmp/kernel
```

Comment:
If the script name contains `.sh` in its name, do we expect to be able to run the script without specifying the whole name (without `.sh`)? Honest question, I googled it and no clear answer. Also, from what it looks like, a god practice is to not include the extension in the name, there's a thread here about this: https://askubuntu.com/questions/764121/sh-specifying-extension.
Anyway, to avoid confusions I would use `make_kernel_image.sh` everywhere or rename it to not include the extension.

Reply:
Fixed!